spec/tasks/seed_devotionals_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+require_relative '../../lib/db_seed_devotionals'
+
+RSpec.describe 'Seeding the database with devotionals', :task do
+  let(:valid_attributes) { attributes_for(:devotional) }
+
+  it 'creates the verses' do
```

Comment:
We've got a copy-paste error here, this should be 'devotionals' not 'verses'.

Reply:
Good catch!